### PR TITLE
[TACHYON-1287] Add Javadoc for start() in LeaderSelectorClient.java

### PR DESCRIPTION
**Pull request by cybermaster**
https://tachyon.atlassian.net/browse/TACHYON-1287

### CODE REVIEW

**Comment by gpang** [comment]
Could you clean up your commit history? It looks like it has commits from another PR. Thanks!

File: common/src/main/java/tachyon/LeaderSelectorClient.java [hunk 1]
```diff
@@ -117,6 +117,11 @@ public void setCurrentMasterThread(Thread currentMasterThread) {
     mCurrentMasterThread = Preconditions.checkNotNull(currentMasterThread);
   }
 
+  /**
+   * Start the Leader selection
```
Comment (yupeng9) [inline common/src/main/java/tachyon/LeaderSelectorClient.java]: NIT: missing dot at the end.

File: common/src/main/java/tachyon/LeaderSelectorClient.java [hunk 2]
```diff
@@ -117,6 +117,11 @@ public void setCurrentMasterThread(Thread currentMasterThread) {
...
+   *
+   * @throws IOException
```
Comment (yupeng9) [inline common/src/main/java/tachyon/LeaderSelectorClient.java]: Can you add docs for this field?

**Comment by yupeng9** [comment]
@cybermaster, thanks for the doc improvement. I'll address the style issues in a followup PR.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**